inc/class-statify-settings.php

```diff
@@ -483,4 +517,27 @@ public static function create_settings_page(): void {
 
 		<?php
 	}
+
+	/**
+	 * Enqueues settings assets.
+	 *
+	 * @return void
+	 */
+	public static function enqueue_settings_assets(): void {
+		wp_enqueue_style(
+			'statify-settings',
+			plugins_url( 'css/settings.css', STATIFY_FILE ),
+			array(),
+			STATIFY_VERSION
+		);
+
+		// Enqueue settings JavaScript.
+		wp_enqueue_script(
+			'statify-settings-js',
```

Comment:
I know we have this for other scripts, but the `-js` suffix is not required here. WordPress handles script and style handle separately and `-js`/`-css` are automatically added to the IDs of the corresponding HTML elements.

```suggestion
			'statify-settings',
```